lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -98,6 +99,29 @@ static FloatAttr getF64FloatAttr(MLIRContext *context, double value) {
   return FloatAttr::get(Float64Type::get(context), value);
 }
 
+static llvm::Optional<Value> getScalarValue(Operation *base_op, Location loc,
+                                            PatternRewriter &rewriter) {
+  llvm::Optional<Value> scalar;
+  if (ValueTensorLiteralOp valueTensorLiteralOp =
+          dyn_cast<ValueTensorLiteralOp>(base_op)) {
+    if (valueTensorLiteralOp &&
+        getTensorRank(valueTensorLiteralOp.getResult()) == 0) {
+      auto val = valueTensorLiteralOp.value()
+                     .cast<ElementsAttr>()
```

Comment:
is this cast necessary?

Reply:
Looks like `.cast<ElementsAttr>()` is not necessary. I removed it.